test/HPE3ParClient_base.py

```diff
@@ -23,6 +23,7 @@
 from testconfig import config
 import datetime
 from functools import wraps
+from hpe3parclient import file_client
```

Comment:
Remove the commented line at line number 28

Reply:
Addressed rework